torch/quantization/_equalize.py

```diff
@@ -0,0 +1,130 @@
+import torch
+import copy
+
+def max_over_ndim(input, axis_list, keepdim=False):
+    ''' Applies 'torch.max' over the given axises
+    '''
+    axis_list.sort(reverse=True)
+    reduced_input = input
+    for axis in axis_list:
+        reduced_input, _ = reduced_input.max(axis, keepdim)
+    return reduced_input
+
+def min_over_ndim(input, axis_list, keepdim=False):
+    ''' Applies 'torch.min' over the given axises
+    '''
+    axis_list.sort(reverse=True)
+    reduced_input = input
+    for axis in axis_list:
+        reduced_input, _ = reduced_input.min(axis, keepdim)
+    return reduced_input
+
+def channel_range(input, axis=0):
+    ''' finds the range of weights associated with a specific channel
+    '''
+    size_of_tensor_dim = len(input.size())
+    mins = input
+    maxs = input
```

Comment:
Same here -- unless you are using it in a loop, and need to initialize the variables, this doesn't really do much